tests/test_manifest.py

```diff
@@ -0,0 +1,46 @@
+"""Test ltdmason.manifest."""
+
+from __future__ import (division, absolute_import, print_function,
+                        unicode_literals)
+from builtins import *  # NOQA
+from future.standard_library import install_aliases
+install_aliases()
+
+import pkg_resources
+import pytest
```

Comment:
When did we decide we are using pytest explicitly (rather than using pytest to drive unittest)? :smile: